package.json

```diff
@@ -225,7 +225,6 @@
     "electron-devtools-installer": "^3.1.1",
     "electron-updater": "^4.3.1",
     "electron-window-state": "^5.0.3",
-    "file-type": "^14.7.1",
```

Comment:
Unused package. Tried to validate for mime-type, but seems doesn't work for `.tgz`